scripts/apps/monitoring/directives/MonitoringGroup.js

```diff
@@ -208,12 +208,18 @@ export function MonitoringGroup(cards, api, authoringWorkspace, $timeout, superd
             scope.$on('refresh:list', (event, group) => {
                 var _viewType = event.currentScope.viewType || '';
 
-                if (group && group._id === scope.group._id || !group &&
-                        _.includes(['highlights',
-                            'spiked',
-                            'single_monitoring',
-                            'monitoring'],
-                        _viewType)) {
+                if (
+                    group
```

Comment:
imo this condition could be a function, would be more readable

Reply:
Would variable work? Fixed in https://github.com/superdesk/superdesk-client-core/pull/2562